packages/utils/tests/react.test.tsx

```diff
@@ -278,12 +278,12 @@ describe("React", () => {
       }
       const { result } = renderHook(() => useAsyncRetry(asyncFunction, []))
       expect(result.current.loading).toBeTruthy()
-      await waitFor(() => {})
-      act(() => {
+
+      act(async () => {
```

Comment:
@setodeve 

* `waitFor()` calls `act()` internally, 
* `result.current.retry()` is a synchronous method.

So you should fix this statements:

```ts
act(() => { result.current.retry() })

await waitFor(() => { expect(result.current.value).toBe("success") })
```